tests/controller/elementControllerTest.js

```diff
@@ -339,7 +437,7 @@ describe('elementController', function () {
                     });
                     expect(res.status).to.be.equal(200);
                     expect(res.body.status).to.be.equal("success");
-                    expect(res.body.results).to.be.equal(2);
+                    expect(res.body.results).to.be.equal(3);
```

Comment:
Why is it 3 ?

Reply:
The number of element that was present during my test. I checked and after each only clean element with price of zero.
![image](https://user-images.githubusercontent.com/52856615/114301391-e7e8e700-9ac4-11eb-863e-76f24f0c4929.png)
Get All takes element that are non active by default. Should we change that ?

Reply:
hmm to discuss, if we change, we'll need a second methode to treat non-active